model/src/main/scala/za/co/absa/atum/model/dto/CheckpointDTO.scala

```diff
@@ -14,15 +14,14 @@
  * limitations under the License.
  */
 
-package za.co.absa.atum.model
-
-import java.time.ZonedDateTime
+package za.co.absa.atum.model.dto
 
 case class CheckpointDTO(
-    name: String,
-    author: String,
-    partitioning: PartitioningDTO,
-    processStartTime: ZonedDateTime,
-    processEndTime: ZonedDateTime,
-    measurements: Seq[MeasurementDTO]
+  id: UUID,
+  name: String,
+  author: String,
+  partitioning: Partitioning,
+  processStartTime: ZonedDateTime,
+  processEndTime: ZonedDateTime,
```

Comment:
Sorry, for another late remark. For eventual streaming, this needs to be 
```suggestion
  processEndTime: Option[ZonedDateTime],
```

Reply:
It makes sense.I'll change it now